src/notifications.rs

```diff
@@ -76,15 +76,15 @@ pub fn notifications_process(
 				.iter_mut()
 				.find(|(k, _v)| k == PANEL_NOTIFICATIONS_FD || k == DAEMON_NOTIFICATIONS_FD)
 			{
-				*v = format!("{}", my_fd.as_raw_fd().to_string());
+				*v = format!("{}", my_fd.as_raw_fd());
```

Comment:
```
warning: `to_string` applied to a type that implements `Display` in `format!` args

   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#to_string_in_format_args
   = note: `#[warn(clippy::to_string_in_format_args)]` on by default
```

Reply:
Hmmm, for this one, I wonder if keeping `.to_string()` but removing the `format!()` is really how this ought to be done...

Reply:
Removing the format would be better.

Reply:
Done